test/unit/app/browser/api/ledgerTest.js

```diff
@@ -135,6 +139,16 @@ describe('ledger api unit tests', function () {
     ledgerClient.returns(lc)
     mockery.registerMock('bat-client', ledgerClient)
 
+    // ledger publisher stubbing
+    const lp = {
+      ruleset: [],
+      getPublisherProps: function (publisher) {
+        return null
+      },
+      Synopsis: batPublisher.Synopsis
+    }
+    mockery.registerMock('bat-publisher', lp)
```

Comment:
This is basically what fixes the tests. Without `bat-publisher` being mocked, I was getting the leak error on my machine